dataGen.js

```diff
@@ -0,0 +1,45 @@
+const fs = require('fs');
+const zlib = require('zlib');
+const faker = require('faker');
+const gzip = zlib.createGzip();
+// const inp = fs.createReadStream('songs.csv');
+// const out = fs.createWriteStream('songs.csv.gz');
+
+const writeSongs = fs.createWriteStream('songs.csv');
+// const compressSongs = gzip(data).pipe(writeSongs);
+writeSongs.write('id,title,album,artist,likes,length\n', 'utf8');
+
+const writeData = (writer, encoding, callback) => {
+  let i = 100;
+  let id = 0;
+  const write = () => {
+    let ok = true;
+    do {
+      i--;
+      id++;
+      const title = faker.lorem.word();
+      const album = faker.lorem.word();
+      const artist = faker.name.lastName();
+      const likes = faker.random.number();
```

Comment:
you might want to think about using a function that can skew the data such that you end up with a lot of songs with a normal amount of likes and a small number of songs with a lot of likes. Lily posted an article about how to do that in helpful links

Reply:
I will add this in a future implementation. For now I need to get my data in both db's.